donate/templates/pages/base_page.html

```diff
@@ -61,22 +61,22 @@
         <div class="footer__nav footer-links">
             <ul class="footer-links__container">
                 <li class="footer-links__item">
-                    <a class="link link--dark-bg" href="https://twitter.com/mozilla" target="_blank" rel="noopener">Twitter</a>
+                    <a class="link link--dark-bg" href="https://twitter.com/mozilla" target="_blank" rel="noopener">{% trans "Twitter" %},</a>
```

Comment:
Looks like you've accidentally added a comma at the end here.

Reply:
oops, good spot!